programs/zstdcli.c

```diff
@@ -708,7 +709,7 @@ int main(int argCount, const char* argv[])
                     if (!strcmp(argument, "--compress-literals")) { literalCompressionMode = ZSTD_lcm_huffman; continue; }
                     if (!strcmp(argument, "--no-compress-literals")) { literalCompressionMode = ZSTD_lcm_uncompressed; continue; }
                     if (!strcmp(argument, "--no-progress")) { FIO_setNoProgress(1); continue; }
-
+                    if (!strcmp(argument, "--exclude-compressed")) { g_excludeCompressedFiles = 1; continue; }
```

Comment:
We have a generic method to set parameters in `fileio` module : 
https://github.com/facebook/zstd/blob/dev/programs/fileio.h#L65 .

It's better to use the same convention.

This will require to create the variable inside `fileio.c`, and expose a helper setter function.

Reply:
Addressed in latest commit.